website_doc/controllers/main.py

```diff
@@ -6,10 +6,7 @@
 from odoo.http import request
 import logging
 import werkzeug.utils
-# from odoo.addons.web import http
-# from odoo.addons.web.http import request
 from odoo.addons.website.controllers.main import Website as controllers
-# from odoo.addons.website.models.website import slugify
 
 logger = logging.getLogger(__name__)
```

Comment:
remove not used logger, please remove both logger var and logging import, apply this to all python files in the module

Reply:
Use ingadhoc/patches for now...